charts/internal/shoot-system-components/charts/csi-driver-node/templates/daemonset.yaml

```diff
@@ -17,6 +17,7 @@ spec:
       annotations:
         checksum/secret-cloud-provider-config: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
       labels:
+        node.gardener.cloud/critical-component: "true"
```

Comment:
The label needs to be added to the `DaemonSet` metadata as well now, see https://github.com/gardener/gardener/issues/7117#issuecomment-1425346730.